main.scss

```diff
@@ -23,7 +26,7 @@
 		}
 	}
 
-	@each $theme, $properties in $o-buttons-themes {
+	@each $key, $theme in $o-buttons-themes {
```

Comment:
just out of curiosity, do we not use `$key` anywhere?

Reply:
Guess not! I should be able to remove it

Reply:
Actually looks like this is needed in sass to get key/values. http://sass-lang.com/documentation/file.SASS_REFERENCE.html#each-directive